.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,36 +1,38 @@
 ---
 name: Bug report
-about: Create a report to help us improve
-title: "[BUG]"
-labels: ''
+about: Create a bug report to help us improve
+title: ""
+labels: ["bug"]
 assignees: ''
-
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+#### Describe the bug
+<!-- A clear and concise description of what the bug is. -->
+
+#### To Reproduce
+<!-- Steps that can be taken to reproduce the behaviour -->
 
-**To Reproduce**
-Steps to reproduce the behavior:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+#### Expected behavior
+<!-- A clear and concise description of what you expected to happen. -->
+
+#### Screenshots
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+#### Logs
+<!-- If applicable, add logs to help explain your problem. -->
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+#### Environment
 
-**Logs**
-If applicable, add logs to help explain your problem.
+- tls-certificates-interface library version: <!-- e.g. 1.2 -->
```

Comment:
I think we can write it in a generic way such as related library versions. Otherwise we need to specify all the consumed libraries including observability_libs, prometheus_k8s, .. etc.